web/index.php

```diff
@@ -13,14 +13,10 @@
 
 if ($app->isLocal()) {
     $app['sessionTimeout'] = 3600 * 24; // Extend session time out in local environment
-    if (!$app->isPhpDevServer()) {
-        $app['twigCacheHandler'] = 'memcache';
-        $app['sessionHandler'] = 'datastore'; // use datastore for dev_appserver local environment
-    }
 } else {
     $app['sessionHandler'] = 'datastore';
-    $app['cacheDirectory'] = sys_get_temp_dir();
     $app['twigCacheHandler'] = 'file';
+    $app['twigCacheDirectory'] = realpath(__DIR__ . '/../cache') . '/twig';
```

Comment:
Noting that this revert is because the Twig files are compiled/cached locally and then sent up as part of the deployment process. We're not actually ever attempting to write to this folder in a running app (which wouldn't be allowed).